### PR TITLE
Cleaning up example actions and adding common ones (Beebole team)

### PR DESCRIPTION
**Pull request by miguel-beebole**
## WHAT

Removed the following actions
- create-company
- update-company

Added most commonly used actions
- run-report
- list-untimed-employees

Updated action list-companies

copilot:summary

copilot:poem

## WHY

The existing services were not complete and rarely used

## HOW

copilot:walkthrough

### CODE REVIEW

**Comment by pipedream-component-development** [comment]
Thank you so much for submitting this! We've added it to our backlog to review, and our team has been notified.

**Comment by pipedream-component-development** [comment]
Thanks for submitting this PR! When we review PRs, we follow the [Pipedream component guidelines](https://pipedream.com/docs/components/guidelines/). If you're not familiar, here's a quick checklist:
- [ ] [Create components to address specific use cases whenever possible](https://pipedream.com/docs/components/guidelines/#component-scope)
- [ ] [Component `key`s should follow the format `app_name_slug-slugified-component-name`](https://pipedream.com/docs/components/guidelines/#component-key-pattern)
- [ ] Components should follow the [standard directory structure](https://pipedream.com/docs/components/guidelines/#folder-structure)
- [ ] [Prefer Node.js client libraries to REST APIs](https://pipedream.com/docs/components/guidelines/#using-apis-vs-client-libraries)
- [ ] When making API requests, [handle pagination](https://pipedream.com/docs/components/guidelines/#pagination) to ensure all data / events are processed
- [ ] [Use `secret` props](https://pipedream.com/docs/components/guidelines/#capturing-sensitive-data) to capture sensitive data
- [ ] Props and methods should be defined in [app files](https://pipedream.com/docs/components/guidelines/#app-files) whenever possible
- [ ] Document methods with [JS Docs](https://pipedream.com/docs/components/guidelines/#methods)
- [ ] [Use `optional` props](https://pipedream.com/docs/components/guidelines/#optional-vs-required-props) whenever possible, and [set a `default` value](https://pipedream.com/docs/components/guidelines/#default-values) where you can
- [ ] Use [async options](https://pipedream.com/docs/components/guidelines/#async-options) to accept user input wherever possible

File: components/beebole_app/actions/run-report/run-report.mjs [hunk 1]
```diff
@@ -0,0 +1,348 @@
...
+      propDefinition: [
+        app,
+        "endda",
+      ],
+    },
+    keys: {
+      type: "string[]",
+      label: "Columns",
+      description: "Select the columns to display in the report.",
+      options: [
```
Comment (GTFalcao) [inline components/beebole_app/actions/run-report/run-report.mjs:27]: Ideally we'd keep these options in a separate file (e.g. `beebole_app/common/constants.mjs`) to keep the action code clean, and import them here

**Review by GTFalcao — CHANGES_REQUESTED** [review]
Hi @miguel-beebole , thanks for the contribution!

I left one comment regarding keeping the code clean. One other thing we need to do is bump the `package.json`'s minor version to reflect this update, e.g. from `0.1.1` to `0.2.0`.

I'll move this forward for now so it can be validated.

**Comment by miguel-beebole** [comment]
Thanks @GTFalcao 

I have corrected the issues with the versioning. Thanks for suggesting the _constants_ change. I will keep that in mind for a future update, if we need to define global constants

**Review by GTFalcao — APPROVED** [review]

**Pull request merged**